### PR TITLE
Add a report a bug link to the navigation

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add a report a bug link to the bottom of the navigation

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check the Report a bug link in the bottom of the navigation works

## Details
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1075

### CODE REVIEW

**Comment by hatched** [comment]
Because the link is `target blank` and a different domain shouldn't it have the 'external' icon? cc @spencerbygraves

**Comment by barrymcgee** [comment]
You need to accommodate for when the nav is in a collapsed state on the model detail pages.

Screenshot 2020-01-13 at 16 07 59

**Comment by spencerbygraves** [comment]
@anthonydillon looks good 👍

**Comment by anthonydillon** [comment]
@barrymcgee good spot, ready for another review.

File: src/static/images/icons/bug-icon.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hatched) [inline src/static/images/icons/bug-icon.svg]: I think you'll want to run this through svgo as I don't think we do it in our build steps.

Comment (anthonydillon) [reply src/static/images/icons/bug-icon.svg]: Good shout, done.

**Comment by barrymcgee** [comment]
Nice icon!

**Pull request merged**